tests/test_visitors/test_tokenize/test_primitives/test_numbers/test_underscored_numbers.py

```diff
@@ -44,7 +52,24 @@ def test_underscored_number(
         '1000.0',
         '-333555',
         '-333555.5',
-        '"10_00"',
+        '1_000',
+        '1_000.0',
+        '+1_000',
+        '+1_000.0',
+        '-1_000',
+        '-1_000.0',
+        '10_000',
+        '100_000',
+        '100_000_000',
+        '"1_00"',
+        '"10_000"',
+        '"100_000"',
+        '"-100_000"',
+        '"1.0_0"',
+        '"3_3.3"',
+        '"1_0_0"',
+        '"1_0_0_1"',
```

Comment:
These numbers are not correct. Number can only considered correct if after split on `_`:
- First group has length of `1`, `2` or `3`
- All other groups have length of strictly `3` 

The same with `float` numbers: we split on `.` and apply this rule to the `float` part as well:
- `0.0_005` is correct
- `100_000.000_001` is correct
- `0.00_05` is not correct

Reply:
I will take part in your discussion.

is the number "1.0_0" considered a number?
or should it be interpreted as a string value, due to the fact used that there are quotes in quotes.


I was earlier to see a used shortened SemVer, an example: `version = "2.4_153"`.
and it was just a string field.

Reply:
Oh, sorry. I missed these quotes. Please, add a comment that you also test strings.